build-package.sh

```diff
@@ -63,6 +63,14 @@ fi
 # Ref: https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+# Import local copy of Travis timing / folding functions (this enables us to
+# produce Travis folded output from within a Docker container)
+# Source: https://github.com/travis-ci/travis-build/tree/master/lib/travis/build/bash
+source ${SCRIPT_DIR}/lib/travis_fold.bash
```

Comment:
Hmm, these are going to get out of date. Could we `curl` them from GitHub directly?

Reply:
We could, though it doesn't look like this code changes (eg: it was introduced 5 years ago, although it's been moved around as their project has been restructured): https://github.com/travis-ci/travis-build/blame/d5760f7925d62fcad1737c02dc6486387aa8b432/lib/travis/build/templates/header.sh#L292

Reply:
My concern with curling them from GitHub is that if they are likely to move around (due to restructuring) then we might break for that reason.  Personally I'd rather live with the risk that the feature stops working in the future.